Dockerfile.multistage

```diff
@@ -0,0 +1,28 @@
+# An example using multi-stage image builds
+
+# First, build the application in the `/app` directory
```

Comment:
I'd anchor this more on the motivation:
1. Install dependencies, which rarely change, so that they get cached in a separate layer.
2. Install the project, which regularly changes
3. Copy to the final, which doesn't have any install tools or cache.

Reply:
This is a duplicate of the contents of `Dockerfile`, hence the lack of exposition.